src/dataStrategies.js

```diff
@@ -20,27 +20,35 @@ export const results = ({
   sortField,
   sortDir,
   highlight,
+  scroll
 }) => {
-  let formatTree = ({ pageSize, page, scrollId }) => ({
-    ..._.pick(['schema', 'join'], tree),
-    type: 'group',
-    key: 'limitedSearchRoot',
-    children: [
-      setFilterOnly(tree),
-      {
-        key: 'results',
-        type: 'results',
-        pageSize,
-        page,
-        include,
-        sortField,
-        sortDir,
-        scroll: true,
-        scrollId,
-        highlight,
-      },
-    ],
-  })
+  let formatTree = ({ pageSize, page, scrollId }) => {
+    let resultsConfig = {
+      key: 'results',
+      type: 'results',
+      pageSize,
+      page,
+      include,
+      sortField,
+      sortDir,
+      highlight,
+    }
+
+    if (scroll !== false) {
```

Comment:
Let's use `if (scroll)`. Please make a PR in bid-search adding this property to the places where we're using this strategy. I can help

Reply:
@sadasant no problem -- smartprocure/bid-search/issues/1269 won't be closed until that work gets done. I'll ping you on the PR for it

Reply:
@sadasant your change would potentially change the behavior of existing uses of the results context while mine wouldn't -- isn't it a lot safer to look for `false` being set?